doc-assets/node/18005-search-videos/18005.html

```diff
@@ -494,92 +414,51 @@ <h3>Open ended ranges</h3>
 
     <h3><code>NOW</code> operator for schedule dates</h3>
 
-    <p>For <code>schedule.starts_at</code> and <code>schedule.ends_at</code>, you can use <code>NOW</code> as a date value. This is a convenience operator that allows you to set up a dynamic query based on the current date-time. By using the exclude option (-), you can also search for videos that have no explicit schedule set. A couple of examples:</p>
+    <p>For <code>schedule.starts_at</code> and <code>schedule.ends_at</code>, you can use <code>NOW</code> as a date value. This is a convenience operator that allows you to set up a dynamic query based on the current date-time.  A couple of examples:</p>
```

Comment:
Removed all the referenced to exclusion.  But maybe we need an example of querying for unscheduled videos?